liblangutil/CharStream.h

```diff
@@ -138,6 +141,7 @@ class CharStream
 private:
 	std::string m_source;
 	std::string m_name;
+	bool m_imported_from_AST;
```

Comment:
`bool m_imported_from_AST = false;`
or 
`bool m_imported_from_AST{false};`

would be better here (similar to the var below)